AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/Helpers/ConfigurationHelper.swift

```diff
@@ -281,11 +281,10 @@ struct ConfigurationHelper {
                  "verificationMechanism": .array(verificationMechanisms)])
         }
 
-        return JSONValue.object(
-            ["auth": .object(
-                ["plugins": .object(
-                    ["awsCognitoAuthPlugin": .object(
-                        ["Auth": .object(
-                            ["Default": authConfigObject])])])])])
+        return JSONValue.object([
+            "Auth": .object([
+                "Default": authConfigObject
+            ])
```

Comment:
ConfigurationHelperTests.testCreateUserPoolJsonConfiguration needs to be updated